scripts/core/registry.yml

```diff
@@ -592,6 +592,12 @@ etors:
 - name: ENQUEUE_NATIVE_COMMAND_EXP
   desc: Enumerator for $xEnqueueNativeCommandExp
   value: '228'
+- name: TENSOR_MAP_ENCODE_IM_2_COL_EXP
+  desc: Enumerator for $xTensorMapEncodeIm2ColExp
+  value: '231'
```

Comment:
If this is where everything's generated from, is this why we're seeing enumerators skip a few values? Should these be `229` and `230`?

Reply:
This `registry.yml` is itself autogenerated. I don't know how to modify this number. Maybe @kbenzie has an idea?

Reply:
The script will only assign a number automatically if the function is absent from `registry.yml`. You can always add it manually.

Reply:
Aha interesting. Thanks @pbalcer

Reply:
Have updated @frasercrmck

Reply:
Another option is to delete the entry from the registry and run the `generate` target again.

Reply:
Interesting, thanks